package.json

```diff
@@ -9,10 +9,9 @@
 		"package": "npx lerna run package",
 		"install": "npx lerna exec -- npm install"
 	},
-	"dependencies": {},
 	"devDependencies": {
-		"lerna": "8.1.2",
-		"rimraf": "5.0.5"
+		"lerna": "^8.1.8",
```

Comment:
These versions aren't fixed and will likely cause unexpected breakages as versions become available and shift.

Reply:
fixing